### DISCUSSION TITLE
CFEngine 3.18 LTS - Extensibility

### DISCUSSION THREAD

**Discussion by olehermanse** [cfengine/core#4723] [General]
We just released a new LTS version, 3.18.0:

https://cfengine.com/blog/2021/cfengine-3-18-lts-released-extensibility/

If you have any feedback or questions about the new features and changes, feel free to comment on this thread 🚀

**Comment by amousset** [comment]
:clap: I've just updated our next major branch from 3.15 to 3.18, we'll test it in the following weeks. I'm particularly exited about the custom promise types, I'm sure it will be of great use!
